### ISSUE TITLE
Incorrect behavior when attempting to add multiedges to a non-multigraph.

### ISSUE THREAD

**Issue by sergeny** [Engelberg/ubergraph#11]
Bug: a nil edge attribute appears from nowhere when attempting to add parallel edges to a non-multi-undirected graph

```
(:attrs (ubergraph/add-edges (ubergraph/graph "a" "b") ["a" "b"] ["b" "a"]))
=>
{#ubergraph.core.UndirectedEdge{:id #uuid"6d53b586-265f-41f3-8462-4e7ebfaeac1e", :src "b", :dest "a", :mirror? true} nil}
```

**Comment by sergeny** [comment]
Note that this is not even a correct way of recording the attribute: the uuid should be the key, not the entire edge object. Consider this:

```
(def G (ubergraph/add-edges (ubergraph/graph "a" "b") ["a" "b"] ["b" "a"]))
(ubergraph/attrs G "a" "b")
=> {}
(ubergraph/attrs G "b" "a")
=> {}
(ubergraph/attrs (ubergraph/set-attrs G ["a" "b"] {:some :attribute}) "a" "b")
=> {:some :attribute}
(:attrs (ubergraph/set-attrs G ["a" "b"] {:some :attribute}))
=>
{#ubergraph.core.UndirectedEdge{:id #uuid"06d9a4a9-9162-4837-86e1-5e88990e0075", :src "b", :dest "a", :mirror? true} nil,
 #uuid"06d9a4a9-9162-4837-86e1-5e88990e0075" {:some :attribute}}
```

**Comment by Engelberg** [comment]
Fixed in 0.1.8
